content/en/security/upcoming_changes_notification_rules.md

```diff
@@ -0,0 +1,63 @@
+---
+title: Upcoming Changes to Security Notification Rules
+kind: documentation
+is_beta: true
+further_reading:
+- link: "/security/notifications/rules/"
+  tag: "Documentation"
+  text: "Notifications Overview"
+- link: "/security/notifications/"
+  tag: "Documentation"
+  text: "Notification Rules"
+---
+
+This article outlines upcoming changes to how [notification rules][1] are configured. Although the most significant changes apply to [Cloud Security Management (CSM)][4], they also impact [Application Security Management][5] and [Cloud SIEM][6].
+
+## Signals deprecation for CSM Misconfigurations
+
+Currently, notifications for [CSM Misconfigurations][2] can only be configured for detection rules that have signals enabled. As part of the upcoming changes to notification rules, you will no longer be required to enable signals in order to generate notifications.
+
+This change will have the following impact on how notifications are generated for CSM Misconfigurations:
+
+1. You will now be able to specify misconfiguration as a source type when creating notification rules.
+1. Signals will no longer be generated for CSM Misconfigurations. This also means that notifications can no longer be enabled for individual detection rules.
+1. Support for CSM Misconfigurations signals will be deprecated in late 2024. Legacy signals will be retained for 15 months from their trigger date (free of charge).
```

Comment:
Is this the product name?
```suggestion
1. Support for CSM Misconfiguration signals will be deprecated in late 2024. Legacy signals will be retained for 15 months from their trigger date (free of charge).
```

Reply:
Yes, in this context it's a "sub-product" of CSM. 

Thanks again for reviewing the PR!